lib/segment/src/index/field_index/full_text_index/mmap_inverted_index/mmap_postings.rs

```diff
@@ -121,7 +122,7 @@ impl MmapPostings {
             chunks,
             data,
             remainder_postings,
-            hw_counter,
+            ConditionedCounter::always(hw_counter),
```

Comment:
This shoudl depend on `populate` parameter on initialization

Reply:
Good catch, thanks!